packages/calcite-components/src/components/alert/alert.tsx

```diff
@@ -203,6 +203,12 @@ export class Alert implements OpenCloseComponent, LoadableComponent, T9nComponen
   }
 
   disconnectedCallback(): void {
+    window.dispatchEvent(
```

Comment:
Instead of having to do this it might be nice if we had a register of alert elements on the alert instance. Should we update this pattern? Would we need an epic to do it in other components as well?

Reply:
https://github.com/Esri/calcite-components/issues/7200